scripts/analyse.sh

```diff
@@ -23,7 +25,8 @@ JSON=$( jq -n \
     '{
         tmsp: $tmsp,
         size: $size,
-        prop: $prop,
+        set_code_prop: $set_code_prop,
```

Comment:
Ok I would say let's stick with having the new line in. I reverted the change -> https://github.com/paritytech/srtool/pull/49/commits/1bfd1e659994fedb07415d8c9768569fd52c04e5

Reply:
We could keep your renaming of `PROP` into `SET_CODE_PROP`, this is nice IMO.
Same around all you introduced, the naming is nicer, I would then only need `prop` to be still availble for now.

Reply:
I don't like duplication but breaking compat. is not an option. If we do break compat. it would be good to wait for a change or a set of changes that are worth it.

I do like your renaming but I think it may not be worth the "troubles" in the end.

The duplication would not remain on the long run but the more I think about it and the more I prefer to keep the legacy name even if this is less accurate with your change.

Reply:
Yes, I agree on rather leaving the old name, and maybe make a ticket (or whatever you use to track things) to rename it in the next breaking change.